applications/tari_base_node/windows/config/log4rs.yml

```diff
@@ -0,0 +1,147 @@
+# A sample log configuration file for running in release mode. By default, this configuration splits up log messages to
```

Comment:
This is not Windows specific. `common/config/logging` carries sample log configs. These should stay in one place so that we don't maintain multiple copies of things.

Reply:
Removed